kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -59,23 +59,54 @@ define([
         CELL = 'cell',
         JOB = 'jobId',
         RESULT = 'result',
+        PING = 'ping',
+        JOB_ID = 'job_id',
+        JOB_ID_LIST = 'job_id_list',
         // messages to be sent to backend for job request types
-        JOB_REQUESTS = {
+        REQUESTS = {
```

Comment:
Renamed these - now they use `REQUESTS` / `RESPONSES` for frontend stuff and `BACKEND_REQUESTS` / `BACKEND_RESPONSES` for the backend. All message types are captured here.